### PR TITLE
Use a standard parameter from the standard definitions.

### PR DESCRIPTION
**Pull request by ioggstream**
## I expect

that common arguments should be in the definitions and/or referenced via $ref

### CODE REVIEW

File: api/public_api_v1.yaml [hunk 1]
```diff
@@ -241,12 +241,7 @@ definitions:
 responses: {}
 parameters:
   PaginationRequest:
-    name: cursor
-    in: query
-    type: string
-    minimum: 1
-    description: An opaque identifier that points to the next item in the collection.
-    x-example: 01BX9NSMKVXXS5PSP2FATZM123
+    $ref: 'https://raw.githubusercontent.com/ioggstream/openapi-samples/master/openapi-v2/schema/components.yaml#/x-components/parameters/cursor'
```
Comment (cloudify) [inline api/public_api_v1.yaml]: instead of `master` you should ref a specific commit, or better a specific tag

**Comment by cloudify** [comment]
@gunzip what do you think? is this going to break our model generation script?

File: api/public_api_v1.yaml (hunk 1 above)
Comment (cloudify) [reply api/public_api_v1.yaml]: also, you should reference an official org (e.g. `teamdigitale` or `italia`), not a personal account

Comment (gunzip) [reply api/public_api_v1.yaml]: AFAIK think this PR is a just a "memento" :) I'd integrate the new interoperability specs as a whole 
once they're stable.

**Comment by ioggstream** [comment]
Close and reopen with a comprehensive patch.

**Pull request closed without merging**